README.md

```diff
@@ -2,6 +2,12 @@
 
 **TODO: Add description**
 
+# Usage
+
+Install dependencies with `mix deps.get`.
+
+Run `mix run --no-halt` to start the bot.
+
```

Comment:
Should also add `$ iex -S mix`. Gives you an interactive REPL in the context of the loaded app.